www.foia.gov/_sass/modules/_misc.scss

```diff
@@ -19,3 +19,13 @@ ul.list-of-exemptions > li > ul{
 .usa-header-extended{
   border-bottom: 1px solid #DEF0FF !important;
 }
+
+.video {
+  width: 640px;
+  height:360px;
+  @media (max-width: 1180px)
```

Comment:
@ameshkin What does the 1180 correspond to? I was thinking it could be smaller, like 700, to match the max width of the paragraphs around it.